src/main.rs

```diff
@@ -83,6 +81,15 @@ fn main() {
         )
         .get_matches();
 
+    let favour = if matches.is_present("stdout") {
```

Comment:
Just a liitle confused. Why not use `micro_service_init`?
This is not a question about this pr.